src/Utils.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <string>
+#include <string.h>
```

Comment:
use `#include <cstring>` instead, that way you can use namespaced versions of the c functionality ...